text/103-leaving-x-twitter.md

```diff
@@ -0,0 +1,68 @@
+   - Supplement the X/Twitter announcement with parallel communications on Discourse, Coq-Club, and team members' personal social media accounts (e.g., Mastodon, LinkedIn).
+   - Specify interest in alternative platforms such as Mastodon, Bluesky, and LinkedIn, but clearly state that the list of platforms is not fixed and remains open to community input.
+   - Emphasize that these platforms will only be adopted if motivated volunteers step forward to manage them.
+
+2. **Suspend activity and redirect** :
+   - After the announcement, cease all activity on X/Twitter, except for possibly updating the profile description to direct followers to new platforms.
+   - In particular, the account will not be used anymore to like, retweet, reply, or post new content.
+
+3. **Create new accounts and animate social media team** :
+   - Create accounts for the Rocq Prover on alternative platforms as motivated volunteers emerge. The accounts can be created before the Rocq Prover renaming is finalized, but their owners should wait for the new visual identity and the go-ahead from the core team before starting to post.
+   - Create a private Zulip channel for coordination between the social media team members.
+
+## Drawbacks
+
+- **Loss of reach and visibility:**
+  Leaving X/Twitter may reduce the project's visibility and reach, especially in the short term. However, this is mitigated by the fact that the project's target audience is increasingly moving to other platforms.
+
+- **Risk of confusion between the messages of renaming and leaving X/Twitter:**
+  Announcing the departure from X/Twitter at the same time as the renaming may confuse the community. However, the renaming was decided in particular because of the inclusivity issues that the previous name raised, and given the openness values that the project wants to promote, remaining on X/Twitter would be contradictory.
+
+## Alternatives
+
+- **Gradual Transition with Auto-Reposting**
+  During a recent Coq Call, an alternative was proposed to maintain the X/Twitter account with auto-reposting from Mastodon and to archive it once the scientific community has largely left X.
+
+  This approach has drawbacks, including: 
+     - Continuing implicit endorsement of X/Twitter.
+     - Requring to keep managing the account, including renaming it, monitoring replies and DMs.
+
+- **Immediate Archiving Without Announcement**
+  Another option is to quietly stop posting on X/Twitter without a formal announcement. However, this could be confusing if the CoqLang account still exists and is never renamed.
+
+## Unresolved questions
+
+- **Which platforms will be chosen for the Rocq Prover social media presence?** This is to be decided after the decision to leave X/Twitter is approved and motivated volunteers step forward.
```

Comment:
Is this ordering of decisions important, and is it already decided privately? I was surprised not to see anything alternatives earlier in the CEP. When opening it, I was expecting discussion that explicitly compared the merits and demerits of the current platform vs potential alternatives. Further, simultaneous presence on some combinations of platforms may be much easier to maintain than on others.

Reply:
nothing has been decided

Reply:
If a decision had been taken, there would not be a need for a CEP / RFC.

> I was surprised not to see anything onalternatives earlier in the CEP.

I propose a method for choosing alternatives (motivated volunteers available) and a few candidate platforms (Mastodon, Bluesky and LinkedIn) though.